### PR TITLE
Update th.m3u

### PR DESCRIPTION
**Pull request by Soponwit19997**

### CODE REVIEW

File: streams/th.m3u [hunk 1]
```diff
@@ -51,43 +57,67 @@ https://cdn-live.dltv.ac.th/dltv15.m3u8
...
+#EXTINF:-1 tvg-id="JKN18.th",JKN 18 (1080p)
+https://edge6a.v2h-cdn.com/jkn18/jkn18.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="LidoChannel.th",Lido Channel (720p)
 https://n-edge-1-th.v2h-cdn.com/lido/lido/playlist.m3u8
 #EXTINF:-1 tvg-id="Mangorn.th",Mangorn (720p)
 http://49.0.87.24:1936/HDDragon/Dragon/playlist.m3u8
+#EXTINF:-1 tvg-id="MCOTHD.th",MCOT HD (720p)
+https://live-org-01-cdn.mcot.net/mcothd1080p_edge/smil:mcothd1080p.smil/chunklist_w58587999_b2000000.m3u8
+#EXTINF:-1 tvg-id="MONO29.th",MONO 29 (1080p)
+https://streaming.monomax.me/Mono29LiveStream/ngrp:myStream_all/chunklist_b6628000.m3u8
```
Comment (Alstruit) [inline streams/th.m3u:75]:
```suggestion
https://streaming.monomax.me/Mono29LiveStream/ngrp:myStream_all/master.m3u8
```
Adaptive

File: streams/th.m3u [hunk 2]
```diff
@@ -51,43 +57,67 @@ https://cdn-live.dltv.ac.th/dltv15.m3u8
...
 #EXTINF:-1 tvg-id="HappyTV65.th",Happy TV 65 (576p)
 https://n-edge-1-th.v2h-cdn.com/happy/happy/playlist.m3u8
+#EXTINF:-1 tvg-id="JKN18.th",JKN 18 (1080p)
+https://edge6a.v2h-cdn.com/jkn18/jkn18.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="LidoChannel.th",Lido Channel (720p)
 https://n-edge-1-th.v2h-cdn.com/lido/lido/playlist.m3u8
 #EXTINF:-1 tvg-id="Mangorn.th",Mangorn (720p)
 http://49.0.87.24:1936/HDDragon/Dragon/playlist.m3u8
+#EXTINF:-1 tvg-id="MCOTHD.th",MCOT HD (720p)
+https://live-org-01-cdn.mcot.net/mcothd1080p_edge/smil:mcothd1080p.smil/chunklist_w58587999_b2000000.m3u8
```
Comment (Alstruit) [inline streams/th.m3u:73]:
```suggestion
https://live-org-01-cdn.mcot.net/mcothd1080p_edge/smil:mcothd1080p.smil/master.m3u8
```
Adaptive

File: streams/th.m3u [hunk 3]
```diff
@@ -3,18 +3,24 @@
...
-#EXTINF:-1 tvg-id="ALTV.th",ALTV (1080p)
-https://thaipbs-ujxrch.cdn.byteark.com/live/playlist_1080p/index.m3u8
 #EXTINF:-1 tvg-id="ALTV.th",ALTV 4 (1080p)
 https://thaipbs-ujxrch.cdn.byteark.com/live/playlist.m3u8
+#EXTINF:-1 tvg-id="AmarinTV.th",Amarin TV (720p)
+https://edge6a.v2h-cdn.com/amarinhd/amarinhd.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="News1.th",ASTV News 1 (720p) [Not 24/7]
 http://news1.live14.com/stream/news1.m3u8
+#EXTINF:-1 tvg-id="Boomerang.th",Boomerang (1080p)
+https://edge6a.v2h-cdn.com/boomerang/boomerang.stream/playlist.m3u8
```
Comment (Alstruit) [inline streams/th.m3u:13]:
```suggestion
```
DMCA

File: streams/th.m3u [hunk 4]
```diff
@@ -51,43 +57,67 @@ https://cdn-live.dltv.ac.th/dltv15.m3u8
...
+https://edge6a.v2h-cdn.com/tnn24/tnn24.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="True4U.th",True 4U (720)
+https://edge6a.v2h-cdn.com/true4u/true4u.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TSports7.th",T Sports 7 (720p)
+https://edge6a.v2h-cdn.com/t_sport/t_sport.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="TVBThai.th",TVB Thai (1080p)
 https://edge6a.v2h-cdn.com/tvb_thai/tvb_thai.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="WhiteChannel.th",White Channel (1080p)
 http://symc-cdn.violin.co.th:1935/tndedge/whitechannel/chunklist.m3u8
+#EXTINF:-1 tvg-id="Workpoint.th",Workpoint 23 (720p)
```
Comment (Alstruit) [inline streams/th.m3u:120]:
```suggestion
#EXTINF:-1 tvg-id="WorkpointTV.th",Workpoint 23 (720p)
```

File: streams/th.m3u [hunk 5]
```diff
@@ -51,43 +57,67 @@ https://cdn-live.dltv.ac.th/dltv15.m3u8
...
+https://streaming.monomax.me/Mono29LiveStream/ngrp:myStream_all/master.m3u8
+#EXTINF:-1 tvg-id="NationTV.th",Nation TV (720p)
+https://edge6a.v2h-cdn.com/nation/nation.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="NBT2HD.th",NBT 2HD (720p)
+https://cdn-edge-ott.prd.go.th/live_vlc/smil:c30f-97f7-c767-ca64-98aa/chunklist.m3u8
 #EXTINF:-1 tvg-id="NBT11Central.th",NBT 11 Central (720p)
 https://cdn-edge-ott.prd.go.th/live_vlc/smil:01f1-8b4b-971e-aa35-d5fa.smil/playlist.m3u8?DVR=
 #EXTINF:-1 tvg-id="NBTWorld.th",NBT World (720p)
 https://cdn-edge-ott.prd.go.th/live_vlc/smil:2609-b4a6-64b3-1431-5e64/chunklist_w507269531_b2128000.m3u8
+#EXTINF:-1 tvg-id="ONE31.th",ONE 31 (1080p)
```
Comment (Alstruit) [inline streams/th.m3u:84]:
```suggestion
#EXTINF:-1 tvg-id="One31.th",ONE 31 (1080p)
```

**Review by Alstruit — DISMISSED** [review]

File: streams/th.m3u [hunk 6]
```diff
@@ -51,43 +55,67 @@ https://cdn-live.dltv.ac.th/dltv15.m3u8
 https://psiiptv-b.sky-cdn.com/ttd20-playlist/index.m3u8
 #EXTINF:-1 tvg-id="ETV.th",ETV (1080p)
 http://150.95.66.20:1935/live/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="GMM25.th",GMM 25 (1080p)
+https://bcovlive-a.akamaihd.net/fb7ac9cc4e1140238e3077ffd29337fe/ap-southeast-1/6415628290001/profile_0/chunklist.m3u8
```
Comment (BellezaEmporium) [inline streams/th.m3u:59]:
```suggestion
https://bcovlive-a.akamaihd.net/fb7ac9cc4e1140238e3077ffd29337fe/ap-southeast-1/6415628290001/playlist.m3u8
```
Adaptive

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/th.m3u [hunk 7]
```diff
@@ -3,18 +3,22 @@
 https://live.x2.co.th/live/13livetv-th.m3u8
 #EXTINF:-1 tvg-id="AtTV.th",@TV (720p)
 http://49.0.87.24:1936/HDAttv/Attv/playlist.m3u8
```
Comment (Dum4G) [inline streams/th.m3u:5]:
```suggestion
http://49.0.87.24:1936/HDAttv/Attv/playlist.m3u8
#EXTINF:-1 tvg-id="ALTV.th",ALTV (1080p)
https://thaipbs-ujxrch.cdn.byteark.com/live/playlist_1080p/index.m3u8
```

**Review by Dum4G — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**